docs/cloud/index.md

```diff
@@ -13,6 +13,17 @@ You can [join the waitlist](https://us17.list-manage.com/survey?u=2334a0f23e55fd
 
 :::
 
+## Upgrade policy
+
+All customers will automatically be upgraded to the latest minor version.
```

Comment:
I don't think we want to guarantee this. Different customers will be upgraded on a different schedule. For example, we can delay the upgrade for some customers to reduce the possibility of outages.